RDKShell/RDKShell.cpp

```diff
@@ -7162,6 +7243,281 @@ namespace WPEFramework {
 
             return result;
         }
+
+        bool RDKShell::MemCheckpointRestoreClient::checkpoint(const std::string &callSign, uint32_t timeouteMs)
```

Comment:
I'm wondering whether we should  move all the RDKShell::MemCheckpointRestoreClient() code to a separate file other than rdkshell.cpp. That file is becoming huge...

Reply:
yes agree, just wonder where MemCheckpointRestore.cpp  should be added, alognside RDKShell.cpp or placed into a separate subfolder ie 3rdParties, or ExtClients